From ivdoorn@gmail.com Sat Mar 21 00:00:57 2009
X-Envelope-From: <linux-wireless-owner@vger.kernel.org>
X-Envelope-To: <vger@sl-h.de>
X-Delivery-Time: 1237590069
X-UID: 447421
Return-Path: <linux-wireless-owner@vger.kernel.org>
X-RZG-CLASS-ID: mi
Received: from vger.kernel.org ([209.132.176.167])
	by mailin.webmailer.de (zeb mi43) (RZmta 18.28)
	with ESMTP id v019d3l2KMUw1q for <vger@sl-h.de>;
	Sat, 21 Mar 2009 00:01:08 +0100 (MET)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1756303AbZCTXBF (ORCPT <rfc822;vger@sl-h.de>);
	Fri, 20 Mar 2009 19:01:05 -0400
Received: (majordomo@vger.kernel.org) by vger.kernel.org id S1756167AbZCTXBF
	(ORCPT <rfc822;linux-wireless-outgoing>);
	Fri, 20 Mar 2009 19:01:05 -0400
Received: from mail-ew0-f165.google.com ([209.85.219.165]:47535 "EHLO
	mail-ew0-f165.google.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
	with ESMTP id S1754535AbZCTXBC (ORCPT
	<rfc822;linux-wireless@vger.kernel.org>);
	Fri, 20 Mar 2009 19:01:02 -0400
Received: by ewy9 with SMTP id 9so894240ewy.37
        for <linux-wireless@vger.kernel.org>; Fri, 20 Mar 2009 16:00:59 -0700 (PDT)
DKIM-Signature:	v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=gmail.com; s=gamma;
        h=domainkey-signature:received:received:from:to:subject:date
         :user-agent:cc:mime-version:content-type:content-transfer-encoding
         :content-disposition:message-id;
        bh=kUUHMEbbgcg3ZzBLae9XtwyEAv+cLzoReox/8uJyJ0g=;
        b=GsQJVlYc+/ZWWWs3SGW9GwMXPrb9BsQWlK9R8go7RckAtZPu7tzKLO/riyrnITPuXy
         1O9cuYRTpuOcAZsddzwpn0dAfesoaoO2tqOdGYcW71oZJ2jTdtshSDv/RaPdbU4TmQMH
         eR8meZJ7bFheAfoZiNwmiRz+Bl+GiUT3DF4rg=
DomainKey-Signature: a=rsa-sha1; c=nofws;
        d=gmail.com; s=gamma;
        h=from:to:subject:date:user-agent:cc:mime-version:content-type
         :content-transfer-encoding:content-disposition:message-id;
        b=uV5UDqv32B//rLgJ8PsaiKFCE+oG6QBYUcY7jogCRSqgWgHR9u8wlVan1ed3KAED5e
         oK/GWNw3OrMmRJjce6t2uGRHd+KOOTNPROjj4uZWwifhfGiT0sgpZ9zynxgiFP9nQvWE
         4r3GmuKOOLJQp89tFTVTo9C3SOTq6n8wNwJaA=
Received: by 10.216.13.209 with SMTP id b59mr1988750web.44.1237590059071;
        Fri, 20 Mar 2009 16:00:59 -0700 (PDT)
Received: from ?192.168.8.39? (hlm-n-2563.mxs.adsl.euronet.nl [212.129.165.99])
        by mx.google.com with ESMTPS id 24sm2795302eyx.38.2009.03.20.16.00.58
        (version=SSLv3 cipher=RC4-MD5);
        Fri, 20 Mar 2009 16:00:58 -0700 (PDT)
From:	Ivo van Doorn <ivdoorn@gmail.com>
To: "John W. Linville" <linville@tuxdriver.com>
Subject: [PATCH] rt2x00: New USB ID for rt73usb
Date:	Sat, 21 Mar 2009 00:00:57 +0100
User-Agent: KMail/1.9.9
Cc: users@rt2x00.serialmonkey.com,
 Xose Vazquez Perez <xose.vazquez@gmail.com>,
 linux-wireless@vger.kernel.org
MIME-Version: 1.0
Content-Type: text/plain;
  charset="utf-8"
Content-Transfer-Encoding: 7bit
Content-Disposition: inline
Message-Id: <200903210000.57321.IvDoorn@gmail.com>
Sender:	linux-wireless-owner@vger.kernel.org
Precedence: bulk
List-ID: <linux-wireless.vger.kernel.org>
X-Mailing-List:	linux-wireless@vger.kernel.org
Status: R
X-Status: NC
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  


Signed-off-by: Ivo van Doorn <IvDoorn@gmail.com>
---
diff --git a/drivers/net/wireless/rt2x00/rt73usb.c b/drivers/net/wireless/rt2x00/rt73usb.c
index 24fdfdf..420fff4 100644
--- a/drivers/net/wireless/rt2x00/rt73usb.c
+++ b/drivers/net/wireless/rt2x00/rt73usb.c
@@ -2425,6 +2425,8 @@ static struct usb_device_id rt73usb_device_table[] = {
 	{ USB_DEVICE(0x0df6, 0x9712), USB_DEVICE_DATA(&rt73usb_ops) },
 	/* Surecom */
 	{ USB_DEVICE(0x0769, 0x31f3), USB_DEVICE_DATA(&rt73usb_ops) },
+	/* Tilgin */
+	{ USB_DEVICE(0x6933, 0x5001), USB_DEVICE_DATA(&rt73usb_ops) },
 	/* Philips */
 	{ USB_DEVICE(0x0471, 0x200a), USB_DEVICE_DATA(&rt73usb_ops) },
 	/* Planex */
--
To unsubscribe from this list: send the line "unsubscribe linux-wireless" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

